plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/frontend.ts

```diff
@@ -190,12 +191,11 @@ const productFiltersStore = {
 				return;
 			}
 
-			const isBlockTheme = getSetting( 'isBlockTheme' );
-			const isProductArchive = getSetting( 'isProductArchive' );
-			const needsRefreshForInteractivityAPI = getSetting(
-				'needsRefreshForInteractivityAPI',
-				false
-			);
+			const sharedSettings = getConfig( 'woocommerce' );
+			const isBlockTheme = sharedSettings?.isBlockTheme || false;
+			const isProductArchive = sharedSettings?.isProductArchive || false;
```

Comment:
`isProductArchive` doesn't belong to shared setting but `woocommerce/product-filters` only.

Reply:
Whoops, good spotting.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/57469/commits/0e589f9657e14d690599c34ca9505fe04c940700